test/test_p4est_3d.jl

```diff
@@ -352,18 +352,18 @@ end
     @test_trixi_include(joinpath(EXAMPLES_DIR,
                                  "elixir_euler_circular_wind_nonconforming.jl"),
                         l2=[
```

Comment:
I did that for the baroclinic instability in Erik's Master's thesis. It can only be compared qualitatively.

Old LMARS implementation:
![oldLMARS](https://github.com/trixi-framework/Trixi.jl/assets/135045760/a400cdbc-e449-4b60-93aa-51ba6d31535a)

New LMARS implementation:
![newLMARS](https://github.com/trixi-framework/Trixi.jl/assets/135045760/94159c33-ec90-42e4-bf5b-e121faff103b)

Reply:
These results look good in the eyeball norm which matches that that changes to the testing reference values occurred after several decimal places.

Reply:
I did not know the eyeball norm before. :laughing: Like it!

Reply:
Nice!